dapps/marketplace/src/pages/transaction/mutations/WithdrawOffer.js

```diff
@@ -114,7 +114,7 @@ class WithdrawOffer extends Component {
           <div className="make-offer-modal">
             <div className="success-icon" />
             <div>
-              <fbt desc="WithdrawOffers.success">Success!</fbt>
+              <fbt desc="success">Success!</fbt>
```

Comment:
Thanks @wanderingstan 🙏 Did we move away from the convention of using the name of the component here?

Reply:
Along with @franckc , we agreed that for short, widely-used strings like "Success!", we just use the word itself as the `desc`. Otherwise translators would see the same string numerous times.

Reply:
+1

Reply:
Oh, I thought CrowdIn was supposed to recognize the string itself and avoid redundant, potentially conflicting translations. 😕

Reply:
If the desc is different, crowdin assumes that the word/phrase must be being used in a different context, and thus creates a separate entry for translators. For things like button titles and boilerplate messages ("Success!") its best to keep them all under one entry, so things are consistent.